support/docker/production/.env

```diff
@@ -20,10 +20,11 @@ PEERTUBE_SMTP_TLS=false
 PEERTUBE_SMTP_DISABLE_STARTTLS=false
 PEERTUBE_ADMIN_EMAIL=<MY EMAIL ADDRESS>
 POSTFIX_myhostname=<MY DOMAIN>
-TRAEFIK_ACME_EMAIL=<MY EMAIL ADDRESS>
-# If you need to obtain ACME certificates for more than one DOMAIN
-# pass them as a comma separated string
-TRAEFIK_ACME_DOMAINS=<MY DOMAIN>
+CERTBOT_EMAIL=<MY EMAIL ADDRESS>
+# Variables to substitute in the nginx template, space-separated
+ENVSUBST_VARS=FQDN PEERTUBE_HOST
+FQDN=<MY DOMAIN>
```

Comment:
What does this even stand for? Why not just `DOMAIN`?

Reply:
That's a fully qualified domain name, but you're right, it should just be `DOMAIN` now.